movie/models.py

```diff
@@ -37,3 +38,14 @@ class Movie(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Comment(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    rating = models.IntegerField()
+    title = models.CharField(max_length=50)
+    date = models.DateTimeField()
+    message = models.CharField(max_length=1000)
```

Comment:
```suggestion
    message = models.CharField(max_length=500)
```

1000 is too much for a comment